### ISSUE TITLE
Question - Physical system description

### ISSUE THREAD

**Issue by muralidn** [liziscool/cas741_project#11]
@liziscool 

It looks like you have omitted the "Physical System Description" section. Wouldn't having a block diagram of the system help establish the problem domain better?

**Comment by liziscool** [comment]
Yes, I struggled with this too but I don't think I could include it. I am not sure I even have a physical system since my program begins with the digital data and I want it to be abstract from where the data came from.

**Comment by smiths** [comment]
I haven't looked at the SRS, but I hope that the "Physical System Description" section hasn't been removed.  Including content that says the section is "not applicable" is much better than removing the section.  We don't want to modify a standard template.

We could consider using abstraction to represent the source of the input data?  Hopefully I'll have some more specific advice to offer once I've looked over the document.  :smile:
